docs/src/main/tut/index.md

```diff
@@ -1,266 +1,3 @@
 ---
-layout: home
-technologies:
- - first: ["Scala", "Fetch is written in Scala and supports both Scala (JVM) and Scala.js (JavaScript environments)"]
- - second: ["Cats", "Fetch uses cats' Free Monad implementation as well as some of its data types."]
- - third: ["Functional Programming", "Fetch is implemented using the Free Monad and Interpreter pattern."]
+layout: fetch-home
```

Comment:
Is this content living beyond this change anywhere?

Is the people responsible for Fetch for Scala aware of this removal?  This would be good to avoid double work on them.

Reply:
I will talk about it with the responsible team

Reply:
Besides, we need to change the features section content since it hasn´t been created yet. That has been token from Nef as example.

Reply:
@purrgrammer communicates me that we can leave out the previous home information since it is a summary of the general documentation